tests/unit/test_clustering.py

```diff
@@ -4,8 +4,19 @@
 def test_kmeans(test_client, test_sample_vector_dataset):
     test_client.vector_tools.cluster.kmeans_cluster(
         dataset_id= test_sample_vector_dataset,
-        vector_fields= ["sample_1_vector_"]
+        vector_fields= ["sample_1_vector_"],
+        overwrite = True
     )
     db_health = test_client.datasets.monitor.health(test_sample_vector_dataset)
     assert '_cluster_' in db_health
-    assert '_cluster_.sample_1_vector_.default' in db_health
+    assert '_cluster_.sample_1_vector_.kmeans' in db_health
```

Comment:
Can we update the kmeans alias to be `kmeans_10` instead? I think this would be really useful as a default compared to just "kmeans"